inf-clojure.el

```diff
@@ -176,13 +178,37 @@ often connecting to a remote REPL process."
 (defcustom inf-clojure-generic-cmd "lein repl"
   "The command used to start a Clojure REPL outside Lein/Boot projects.
 
-Alternative you can specify a TCP connection cons pair, instead
+Alternatively you can specify a TCP connection cons pair, instead
 of command, consisting of a host and port
 number (e.g. (\"localhost\" . 5555)).  That's useful if you're
 often connecting to a remote REPL process."
   :type '(choice (string)
                  (cons string integer)))
 
+(defvar-local inf-clojure-repl-flavor nil
```

Comment:
It is maybe better, I so like flavor I will need to find an open source project where to use it 😀 Only Gradle was so bold as to actually add it as concept 😁